### PR TITLE
Shuffle order of projects on site generation

### PR DESCRIPTION
**Pull request by Cadair**
So all the astropy ones are not at the top ;)

### CODE REVIEW

File: _plugins/shuffle.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Jekyll
```
Comment (dpshelio) [inline _plugins/shuffle.rb]: I'm not sure whether plugins works on github-pages. They have a [list of accepted plugins](https://help.github.com/articles/adding-jekyll-plugins-to-a-github-pages-site/) for some reason...  Probably a js function would be more safe...

**Comment by dpshelio** [comment]
By the way, this only shuffles when it's build, so it could be once... and that's it. Done forever. Till we merge a new PR. So not of much help for what we want.

**Comment by Cadair** [comment]
@dpshelio I can't javascript so this is as good as it gets from me.

**Comment by dpshelio** [comment]
So I'll try to do it sometime this weekend.

**Comment by Cadair** [comment]
we can always revert (not delete the commit, just revert). I am going to merge it and see what happens.

**Comment by Cadair** [comment]
it worked @dpshelio

**Comment by Cadair** [comment]
ok, it didn't work. It also didn't break everything.

**Pull request merged**